docs/source/tutorials/localisation-translations.rst

```diff
@@ -61,7 +61,7 @@ Go to :guilabel:`File` and then :guilabel:`Save as`, and save it with a new name
     .. figure:: tutfigures/conventions.png
         :width: 600
 
-Therefore, the Italian language file to be used for research with the Lombard-Italian bilingual community based in Italy is named: :file:`lmoIta_Ita_IT.json`. 
+    Therefore, the Italian language file to be used for research with the Lombard-Italian bilingual community based in Italy is named: :file:`lmoIta_Ita_IT.json`. 
```

Comment:
While you go over this bit, maybe fix the error where :file:`lmoIta_Ita_IT.json` should be :file:`LmoIta_Ita_IT.json`